### ISSUE TITLE
Error in grid::unit(x$height, "mm"): 'x' and 'units' must have length > 0

### ISSUE THREAD

**Issue by tungttnguyen** [wilkox/ggfittext#26]
Hello,

I recently had this strange error. Can you tell me what I did wrong? 

Thanks!

``` r
library(ggplot2)
library(ggfittext)

df <- data.frame(
  stringsAsFactors = FALSE,
  x = c(10,10,25,25,50,50,100,100,200,
        200,500,500),
  variable = c("AAA","nAAA","AAA","nAAA","AAA",
               "nAAA","AAA","nAAA","AAA","nAAA","AAA",
               "nAAA"),
  value = c(-4.5,-0.1,-1.2,0,-34.1,-1.8,-38.2,
            -2.1,-3.8,-0.1,0,0)
)
df
#>      x variable value
#> 1   10      AAA  -4.5
#> 2   10     nAAA  -0.1
#> 3   25      AAA  -1.2
#> 4   25     nAAA   0.0
#> 5   50      AAA -34.1
#> 6   50     nAAA  -1.8
#> 7  100      AAA -38.2
#> 8  100     nAAA  -2.1
#> 9  200      AAA  -3.8
#> 10 200     nAAA  -0.1
#> 11 500      AAA   0.0
#> 12 500     nAAA   0.0

p0 <- ggplot(df, aes(x = factor(x),
                     y = value,
                     label = value)) +
  facet_wrap(variable ~ .) +
  geom_col() +
  theme_bw()
p0
```

![](https://i.imgur.com/XXpjbKb.png)

### Error

``` r
p0 +
  geom_bar_text()
#> Error in grid::unit(x$height, "mm"): 'x' and 'units' must have length > 0
```

![](https://i.imgur.com/ONHc3Vt.png)

<sup>Created on 2020-11-28 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by wilkox** [comment]
Thanks for reporting this @tungmilan. It looks like it was due to a heuristic for setting the text boundary box which when I dug into it doesn't actually seem to be necessary any more. I've fixed it in the latest dev release (0.9.0.9002) which you can install with `devtools::install_github('wilkox/ggfittext')`.

**Comment by tungttnguyen** [comment]
Working now. Thanks a lot for the quick fix!
